### ISSUE TITLE
yarnpkg failing for version 22

### ISSUE THREAD

**Issue by yu210148** [yu210148/gvm_install#95]
Looks like the `yarnpkg` command is failing on line 361 with:

> yarn install v1.22.19
[1/5] Validating package.json...
error gsa@22.4.1: The engine "node" is incompatible with this module. Expected version ">=14.0". Got "12.22.9"
error Found incompatible module.

I suspect this has to do with that repository it adds into /etc/apt/sources.list.d/yarn.list but I'm not sure yet.

**Comment by yu210148** [comment]
Looks like this may also be affecting line 124/5 `yarn install` & `yarn upgrade` too.

**Comment by yu210148** [comment]
Yes, I believe the issue is with how it's dealing with yarn and node js. Dropping these links with more info:

https://classic.yarnpkg.com/en/docs/cli/install

https://yarnpkg.com/getting-started/install

**Comment by yu210148** [comment]
The instructions at https://greenbone.github.io/docs/latest/22.4/source-build/index.html also suggest it's going to have to build something called pg-gvm

I also see something about exporting a 'NODE_VERSION' in the gsa build instructions there.

**Comment by yu210148** [comment]
Just had someone report that they ran into this on Debian as well.

**Comment by yu210148** [comment]
Hi All, 

I made some progress on this today in the issue95 branch. Still don't know if it works and haven't dealt with this pg-gvm business yet but at least I got it to build the gsa so I thought I'd update this thread.

**Comment by backl1n** [comment]
@yu210148 salut jai le meme probleme as tu trouver une solution ?

**Comment by backl1n** [comment]
je suis sous ubuntu server  20.04

**Comment by yu210148** [comment]
I'm afraid not. I added in some code to build the pg-gvm component then life got in the way and I haven't gotten around to getting any more work done on it.
